docs-markdown/src/test/data/repo/articles/list.md

```diff
@@ -0,0 +1,29 @@
+---
+# required metadata
+title: List
+description: The OPS platform guide to Markdown and DocFX Flavored Markdown (DFM) extensions.
```

Comment:
Might be good to have the description explain that the lists are unformated and are changed by the tests.